dotCMS/src/main/java/com/dotcms/publishing/manifest/ManifestBuilder.java

```diff
@@ -0,0 +1,15 @@
+package com.dotcms.publishing.manifest;
+
+import com.dotcms.publishing.manifest.ManifestItem.ManifestInfo;
+import com.dotmarketing.exception.DotRuntimeException;
+import java.io.Closeable;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public interface ManifestBuilder extends Closeable {
```

Comment:
Implement AutoCloseable in order to use such as

try (ManifestBuilder mb) {

}

Reply:
Closeable extends AutoCloseable